com.unity.render-pipelines.core/ShaderLibrary/CommonLighting.hlsl

```diff
@@ -290,6 +290,41 @@ real GetSpecularOcclusionFromBentAO(real3 V, real3 bentNormalWS, real3 normalWS,
     return SphericalCapIntersectionSolidArea(cosAv, cosAs, cosB) / (TWO_PI * (1.0 - cosAs));
 }
 
+real GetSpecularOcclusionFromBentAO(real3 V, real3 bentNormalWS, real3 normalWS, real ambientOcclusion, real roughness)
+{
+    //SphericalGaussian NDF = WarpedGGXDistribution(normalWS, roughness, V);
```

Comment:
can you add "pseudo code" here just so we don't think those function exist. Thanks